cmd/kubeapps-apis/server/server.go

```diff
@@ -34,16 +35,28 @@ import (
 	pluginsGRPCv1alpha1 "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/core/plugins/v1alpha1"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/reflection"
+	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/encoding/protojson"
 	log "k8s.io/klog/v2"
 )
 
+// LogRequest is a gRPC UnaryServerInterceptor that will log the API call
+func LogRequest(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (response interface{}, err error) {
+	start := time.Now()
+	res, err := handler(ctx, req)
```

Comment:
What I find useful here is having a way to also log the request params. I mean,  don't like handling req params in each operation handler like:
`+core GetAvailablePackageSummaries (cluster="kubeapps", namespace="kubeapps")`

What if we add a more verbose tace (like V4) to also log the request params? WDYT?

Reply:
Yeah, it'd be nice but I think that's difficult because the request message is different for each RPC call, so we can't just pull out the cluster and namespace, for example. At least, I don't see a way to do so without something very ugly like trying to cast the `interface{}` to each type of request message. But maybe you've thought of another way Antonio? Hmm, perhaps we could define an interface for a `func RequestSummary()` that we ensure for each request message and just cast to that... but either way, I think that should be out of scope for this PR (but worth investigating).